src/api/client.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+const client = axios.create({
+    // Later read this URL from an environment variable
+    baseURL: process.env.REACT_APP_API_ENDPOINT || "http://sl.absolutzero.org/api"
```

Comment:
If not in a development environment then use the URL for the production server API endpoint instead.
If in a dev environment use the one defined in `.env.development` file